### PR TITLE
Do not create AggregatorRunner if the corresponding section is missing

### PR DESCRIPTION
**Pull request by Barthelemy**

### CODE REVIEW

File: Framework/src/InfrastructureGenerator.cxx [hunk 1]
```diff
@@ -476,6 +476,11 @@ vector<OutputSpec> InfrastructureGenerator::generateCheckRunners(framework::Work
 void InfrastructureGenerator::generateAggregator(WorkflowSpec& workflow, std::string configurationSource, vector<framework::OutputSpec>& checkRunnerOutputs)
 {
   // TODO consider whether we should recompute checkRunnerOutputs instead of receiving it all baked.
+  auto config = ConfigurationFactory::getConfiguration(configurationSource);
+  if (config->getRecursive("qc").count("aggregators") == 0) {
+    ILOG(Warning, Devel) << "No \"aggregators\" structure found in the config file. If no postprocessing is expected, then it is completely fine." << ENDM;
```
Comment (knopers8) [inline Framework/src/InfrastructureGenerator.cxx:481]: If no aggegators are expected

**Review by knopers8 — APPROVED** [review]
Thanks, I've already felt the problem now ;) Could you just fix the log message please?

**Comment by Barthelemy** [comment]
Yes, this is something you encounter pretty fast :) 
Fixed the message.

**Comment by Barthelemy** [comment]
macos build is broken for another reason, merging.

**Pull request merged**